pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -908,9 +908,32 @@
     dependencies = with self; [ nvim-lspconfig ];
   };
 
-  nvim-spectre = super.nvim-spectre.overrideAttrs {
-    dependencies = with self; [ plenary-nvim ];
-  };
+  nvim-spectre = super.nvim-spectre.overrideAttrs
+    (old:
+      let
+        spectre_oxi = rustPlatform.buildRustPackage
```

Comment:
make it available in passthru in case one want to change it/override it.

Reply:
Good idea !